app/logic/Mail/EMail.ts

```diff
@@ -201,36 +202,11 @@ export class EMail extends Message {
   async removeTagOnServer(tag: Tag) {
   }
 
-  async respondToInvitation(response: InvitationResponseInMessage): Promise<void> {
-    assert(this.invitationMessage == InvitationMessage.Invitation, "Only invitations can be responded to");
-    let event: Event;
-    for (let calendar of appGlobal.calendars) {
-      event = calendar.events.find(event => event.calUID == this.event.calUID);
-      if (event) {
-        break;
-      }
-    }
-    if (!event) {
-      let calendar = appGlobal.calendars.first;
-      event = calendar.newEvent();
-      event.copyFrom(this.event);
-      event.response = InvitationResponse.NoResponseReceived;
-      calendar.events.add(event);
-      if (event.recurrenceRule) {
-        event.fillRecurrences(new Date(Date.now() + 1e11));
-      }
-    }
-    let participant = event.participants.find(participant => participant.emailAddress == this.folder.account.emailAddress);
-    if (participant) {
-      event.response = participant.response = response;
-      await event.save();
-      await this.sendInvitationResponse(response);
-    }
-    /* else add participant? */
-  }
-
-  protected async sendInvitationResponse(response: InvitationResponseInMessage): Promise<void> {
-    await this.folder.account.sendInvitationResponse(this.event, response);
+  getUpdateCalendars(): Collection<Calendar> {
+    assert(this.invitationMessage && this.event, "Must have event to find calendar");
+    let validCalendars = appGlobal.calendars.filter(calendar => !calendar.mainAccount);
```

Comment:
Why this condition that the calendar cannot be related to an email account?

Note that we will have CalDAV accounts that will have an IMAP account as mainAccount.